chromadb/test/test_api.py

```diff
@@ -1,17 +1,20 @@
 # type: ignore
+import os
```

Comment:
As an FYI this file is sort of a cesspool / trash bucket / laundry pile / pick your euphemism of choice. We don't really use it anymore, just exists for legacy reasons. We threw this together in 1-2 days when we made the first version of chroma very, very speedily.